docs/examples.rst

```diff
@@ -141,3 +141,14 @@ One form of running this example could be:
 .. explicit external hyperlink targets
 
 .. _examples: https://github.com/LLNL/UnifyFS/tree/dev/examples/src
+
+Transfer API
+------------
+
+UnifyFS has a transfer API to move files from the UnifyFS file space out to external storage (or from external storage into the UnifyFS file space).  This can be invoked using the "transfer" application in the examples directory.  The transfer source code also serves as a template for using transfer API through C functions.
+
+.. code-block:: Bash
+
+    $ srun -N4 -n4 transfer-static /unifyfs/snapshot.bin /scratch/snaps/snapshot.bin
+
+(assuming that /unifyfs/snapshot.bin is a file you've written within the UnifyFS file space by an application.)
```

Comment:
Perhaps a simpler example might be:
"srun -N4 -n4 transfer-static /unifyfs/file1 /scratch/mydir/file1

where file1 is a file that was previously written and laminated in UnifyFS."

Reply:
What does the C API look like? Can we put the prototype here similarly to what is done for mount/unmount? https://unifyfs.readthedocs.io/en/latest/api-mount.html

Reply:
> Perhaps a simpler example might be:
> "srun -N4 -n4 transfer-static /unifyfs/file1 /scratch/mydir/file1
> 
> where file1 is a file that was previously written and laminated in UnifyFS."

Sounds fine to me.

Reply:
> What does the C API look like? Can we put the prototype here similarly to what is done for mount/unmount? https://unifyfs.readthedocs.io/en/latest/api-mount.html

Certainly, and I will do that.  With the ECP meeting imminent and my having a commitment on Friday late afternoon, I put into the docs the shortest thing I was confident would be correct and informative.

Reply:
Okay, if you don't have time to document the API, can you put a reference to the header file where the API is defined perhaps? E.g. For more information on the transfer API please refer to client/src/unifyfs.h .

Reply:
I have time to today.  I just didn't yesterday afternoon.  I'll work on that as soon as I get the other changes to both work and satisfy the syntax checker.